sdks/python/apache_beam/yaml/standard_io.yaml

```diff
@@ -139,8 +139,11 @@
     mappings:
       'ReadFromCsv':
         path: 'path'
+        delimiter: 'delimiter'
+        comment: 'comment'
```

Comment:
I don't see any changes in the transform provider related to comment, is that intentional?

Reply:
These parameters were in the upstream transform (along with everything else from Pandas); this change simply exposes them.

Reply:
So, now that I had a chance to actually look at this, it is WAI. 

The Python Reader explicitly subsets (via the renaming transform) to comment and delimiter, while the writer supports delimiter (a comment marker specifying lines to ignore doesn't makes sense for a writer). 

There is no Java CSV reader. The writer supports a delimiter (but, as with Python, no need for comment). These are read directly from the provided transform config schema at discovery time. 

Sorry for the confusion.

Reply:
SGTM, LGTM

Reply:
SGTM, LGTM